src/v0/destinations/klaviyo/transform.js

```diff
@@ -275,9 +301,101 @@ const process = async (event) => {
   return result;
 };
 
+const batchEvents = (successRespList) => {
```

Comment:
This function only does batching for subscribe events hence should be named accordingly also the parameter should be named accordingly and need to add reference documentation